diff --git a/src/vitepress/components/VPContentDocOutline.vue b/src/vitepress/components/VPContentDocOutline.vue
index 508e28baad95511db50302e3e7f22d1da824c8d7..e7cf467b9e99ba5d6e586872ca9ce3eaeebe9378 100644
--- a/src/vitepress/components/VPContentDocOutline.vue
+++ b/src/vitepress/components/VPContentDocOutline.vue
@@ -27,10 +27,10 @@ const handleClick = ({ target: el }: Event) => {
 <template>
   <div class="VPContentDocOutline" ref="container">
     <div class="outline-marker" ref="marker" />
-    <div class="outline-title">On this page</div>
+    <div class="outline-title">页面梗概</div>
     <nav aria-labelledby="doc-outline-aria-label">
       <span id="doc-outline-aria-label" class="visually-hidden"
-        >Table of Contents for current page</span
+        >当前页面的目录</span
       >
       <ul class="root">
         <li
diff --git a/src/vitepress/components/VPLocalNav.vue b/src/vitepress/components/VPLocalNav.vue
index d4d144c31ea3c23a8342f39fc64a27c094c90768..c384fe15bc74a767e910aeb84c71e041a11a8210 100644
--- a/src/vitepress/components/VPLocalNav.vue
+++ b/src/vitepress/components/VPLocalNav.vue
@@ -22,7 +22,7 @@ function scrollToTop() {
       @click="$emit('open-menu')"
       >
       <VTIconAlignLeft class="menu-icon" />
-      <span class="menu-text">Menu</span>
+      <span class="menu-text">菜单</span>
     </button>
 
     <a
@@ -30,7 +30,7 @@ function scrollToTop() {
       class="top-link"
       href="#"
       @click="scrollToTop"
-      >Return to top</a
+      >返回顶部</a
     >
   </div>
 </template>
diff --git a/src/vitepress/components/VPNavBar.vue b/src/vitepress/components/VPNavBar.vue
index a09d739a35dfa5ef12ff779cdeab7c0bf1b7000e..0e1acfd82632dc0fc666bb4e97816c90c4d154d8 100644
--- a/src/vitepress/components/VPNavBar.vue
+++ b/src/vitepress/components/VPNavBar.vue
@@ -43,6 +43,7 @@ defineProps<{
   padding: 0 12px 0 24px;
   height: var(--vt-nav-height);
   background-color: var(--vt-c-bg);
+  white-space: nowrap;
   transition: border-color 0.5s, background-color 0.5s;
 }
 
diff --git a/src/vitepress/components/VPNavBarExtra.vue b/src/vitepress/components/VPNavBarExtra.vue
index 5598a50be69e4558c66247eb8fc2c95e24a44c32..513e666c31c836d63334ed308ad5feca179e2b2f 100644
--- a/src/vitepress/components/VPNavBarExtra.vue
+++ b/src/vitepress/components/VPNavBarExtra.vue
@@ -14,7 +14,7 @@ const hasContent = computed(() => {
   <VTFlyout v-if="hasContent" class="VPNavBarExtra" label="extra navigation">
     <div v-if="config.appearance" class="vt-menu-group">
       <div class="vt-menu-item item">
-        <p class="vt-menu-label">Appearance</p>
+        <p class="vt-menu-label">外观</p>
         <div class="vt-menu-action action">
           <VTSwitchAppearance />
         </div>
diff --git a/src/vitepress/components/VPNavBarSearch.vue b/src/vitepress/components/VPNavBarSearch.vue
index 75858ca96a10c0e9d41c789edace838cb1d84504..89767d1e1cb0e7db5ae814eb0e374d4ac66525e1 100644
--- a/src/vitepress/components/VPNavBarSearch.vue
+++ b/src/vitepress/components/VPNavBarSearch.vue
@@ -67,7 +67,7 @@ function load() {
               stroke-linejoin="round"
             ></path>
           </svg>
-          <span class="DocSearch-Button-Placeholder">Search</span>
+          <span class="DocSearch-Button-Placeholder">搜索</span>
         </span>
         <span class="DocSearch-Button-Keys">
           <span class="DocSearch-Button-Key" ref="metaKey">Meta</span>
diff --git a/src/vitepress/components/VPNavScreenAppearance.vue b/src/vitepress/components/VPNavScreenAppearance.vue
index 51714078639dac6add217817e37568293623ec8d..3b776dab2fb032109b68f20032050b446e86c6d8 100644
--- a/src/vitepress/components/VPNavScreenAppearance.vue
+++ b/src/vitepress/components/VPNavScreenAppearance.vue
@@ -7,7 +7,7 @@ const { config } = useConfig()
 
 <template>
   <div v-if="config.appearance" class="VPNavScreenAppearance">
-    <p class="text">Appearance</p>
+    <p class="text">外观</p>
     <VTSwitchAppearance />
   </div>
 </template>
diff --git a/src/vitepress/components/VPNotFound.vue b/src/vitepress/components/VPNotFound.vue
index 32b94c2a98ce3101af33228058caeea0ea0c56d8..984ceaa635a76d940a82f35f63130f9b7677256f 100644
--- a/src/vitepress/components/VPNotFound.vue
+++ b/src/vitepress/components/VPNotFound.vue
@@ -7,16 +7,15 @@ const route = useRoute()
 
 <template>
   <div class="vt-doc">
-    <h1>Page Not Found</h1>
+    <h1>页面未找到</h1>
     <p>
-      You found a dead link: <span class="not-found-path">{{ route.path }}</span
+      你打开了一个不存在的链接：<span class="not-found-path">{{ route.path }}</span
       ><br />
       <span v-if="theme.repo"
-        >Please
+        >不介意的话请提交到
         <a :href="`https://github.com/${theme.repo}`" target="_blank"
-          >let us know</a
-        >
-        so we can fix it.</span
+          >这里</a
+        >，我们会跟进修复。</span
       >
     </p>
   </div>